front/app/containers/SignUpInPage/index.tsx

```diff
@@ -133,7 +133,9 @@ const SignUpPage = ({
   useEffect(() => {
     const subscriptions = [
       openSignUpInModal$.subscribe(({ eventValue: newMetaData }) => {
-        if (newMetaData) setMetaData(newMetaData);
+        if (newMetaData) {
+          setMetaData(newMetaData);
+        }
```

Comment:
Perhaps we can look into removing the `// eslint-disable-next-line react-hooks/exhaustive-deps` comment for this useEffect? These are left-overs from the linter upgrade and we should try to get rid of them little by little.